client/blocks/reader-post-actions/index.jsx

```diff
@@ -86,10 +87,10 @@ const ReaderPostActions = ( props ) => {
 			{ shouldShowComments( post ) && (
 				<li className="reader-post-actions__item">
 					<CommentButton
+						href={ getPostUrl( post ) + '#comments' }
 						key="comment-button"
 						commentCount={ post.discussion.comment_count }
 						onClick={ onCommentClick }
-						tagName="button"
```

Comment:
Why was this changed from a `button` to an `a`? Which one _should_ it be?

Reply:
This was actually prompted by your feedback on #46737:

"Comments: This should be a <a> link to the comments instead of a <div> along with the post title so it can be differentiated from other comment links, such as "[number] Comment(s): [post tile]"

We'll always have an href to the comments section, even if we then employ extra functionality like smoothly scrolling to the right spot. Is `<a>` still the right element?

Reply:
haha - this sounds like me. I recently reviewed my own code not realizing it was my own, wondering why it had been done that way. 

As long as it's linking to a different spot, then an `<a>` is appropriate. If it were to _reveal_ a comments section, like "Show Comments" then a `<button>` may be more appropriate. That's what I'm concerned about with locking in the `a` - are there areas where we would use a button? If not, then an `<a>` is fine and we can refactor if needed later.